base/docs/helpdb.jl

```diff
@@ -2024,13 +2024,6 @@ Reseed the random number generator. If a `seed` is provided, the RNG will give a
 srand
 
 doc"""
-    isexecutable(path) -> Bool
-
-Returns `true` if the current user has permission to execute `path`, `false` otherwise.
```

Comment:
should remove from rst if you're removing this docstring entirely. also regen the rst for the adjustments to isreadable and iswritable docstrings.